index.js

```diff
@@ -40,8 +46,10 @@ const inputs = [
         validator: v => v.match(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i)
     }),
     read({
-        prompt: 'Domain that the cert will be issued for, e.g. example.com:',
-        argument: 'domain'
+        prompt: 'Domain(s) that the cert will be issued for (separated by spaces), e.g. example.com www.example.com foo.example.com:',
+        argument: 'domain',
+        processValue: v => v === undefined || Array.isArray(v) ? v : v.trim().split(/\s+/),
+        validator: v => v !== undefined && v.length > 0 && v.every(item => typeof item === 'string' && item.length > 0)
```

Comment:
The `processValue` is needed, because the user may choose to not pass any arguments and enter them when asked to. So `yargs` gives us an array but, in the latter case (user enters values one by one) we get a single string that includes domains separated by space.

The validation of presence is also needed: If you pass `--domain` (with no values) `yargs` creates an empty array for `domain`.